src/Buttons.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Buttons from './Buttons';
+
+describe('Buttons', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const handleClick = jest.fn();
+
+  function renderButtons({ buttons, selectId } = { selectId: null }) {
```

Comment:
`default` 값을 처리한거라면 아래처럼 해줘도 되지 않을까요?
```suggestion
  function renderButtons({ buttons, selectId = null }) {
```